VideoLocker/res/layout/fragment_course_dashboard.xml

```diff
@@ -49,7 +50,7 @@
     <View
         android:layout_width="match_parent"
         android:layout_height="1dp"
-        android:background="@android:color/darker_gray" />
+        android:background="@color/edx_grayscale_neutral_light" />
```

Comment:
Should this be moved to styles too? There is already a base separator style and an implementation of it present.